### ISSUE TITLE
Quick Stack Nearby Broken

### ISSUE THREAD

**Issue by CoderCow** [CoderCow/Protector-Plugin#47]
Causes items to get lost or bank chest not to update properly or not to work at all.

**Comment by antagonist99** [comment]
With this [new hook (Packet 85) ](https://tshock.readme.io/docs/multiplayer-packet-structure#section-force-item-into-nearest-chest-85-)is there any possibility of changing the current code to work? 

Quick Stack Nearby is really a very useful function, even if it does not initially work with safe chests. 

Thank very much for keeping this project active.

**Comment by CoderCow** [comment]
Yes, it is possible and Protector is actually using this packet already. The distribution of the items to be stacked is just buggy. Though it will require a decent amount of time to fix and test this.

**Comment by PBXGuy** [comment]
Hello, I'm a new user of TShock and your plugin.  Is there any update to getting Quick Stack Nearby to work?

**Comment by CoderCow** [comment]
Nope, because neither me nor other people have shown much interest in getting this fixed. It actually looks like most players are fine with managing their items manually.

**Comment by PBXGuy** [comment]
Hmm, I guess I'm just lazy. :D  Okay, thanks for the reply anyway!
